.travis.yml

```diff
@@ -14,7 +14,7 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean check -x checkstyleTest --stacktrace
+  - ./gradlew clean check -x checkstyleTest -x :apollo-gradle-plugin:test --stacktrace
```

Comment:
check should be build otherwise there is a chance that tests pass but the assembly of the aar will fail.